tools/atip/atip/android/android.py

```diff
@@ -116,18 +117,22 @@ def quit(self):
 
 
     def openSettings(self):
-        self.doCMD(self.adb + " am force-stop com.android.settings")
-        settings_cmd = self.adb + \
-                    " am start -n " + \
-                    "com.android.settings/.Settings"
-        try:
-            (return_code, output) = self.doCMD(settings_cmd)
-            if return_code == 0:
-                if self.d.info["currentPackageName"] == "com.android.settings":
-                    return True
-            return False
-        except Exception as e:
-            return False
+        #Open settings via command line does not work on Xiaomi Pad2
+        if self.productName != PRODUCTS_NAME["xiaomipad2"]:
```

Comment:
prefer to write as：

if self.productName == PRODUCTS_NAME["xiaomipad2"]:
.....
else:
....